proposals/description/cr_oracle_cleanup.ts

```diff
@@ -0,0 +1,385 @@
+      description: 'Revoke SWAP_ADMIN_ROLE from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('FUSE_ADMIN'), addresses.optimisticTimelock],
+      description: 'Revoke FUSE_ADMIN from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_GUARDIAN_ADMIN_ROLE'), addresses.optimisticTimelock],
+      description: 'Revoke PCV_GUARDIAN_ADMIN_ROLE from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('METAGOVERNANCE_GAUGE_ADMIN'), addresses.optimisticTimelock],
+      description: 'Revoke METAGOVERNANCE_GAUGE_ADMIN from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('PCV_MINOR_PARAM_ROLE'), addresses.optimisticTimelock],
+      description: 'Revoke PCV_MINOR_PARAM_ROLE from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeRole(bytes32,address)',
+      arguments: (addresses) => [ethers.utils.id('TOKEMAK_DEPOSIT_ADMIN_ROLE'), addresses.optimisticTimelock],
+      description: 'Revoke TOKEMAK_DEPOSIT_ADMIN_ROLE from optimisticTimelock (OA is deprecated)'
+    },
+    {
+      target: 'tribeReserveStabilizer',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in Tribe Reserve Stabilizer (backstop) to the uncached CR Oracle'
+    },
+    {
+      target: 'pcvEquityMinter',
+      values: '0',
+      method: 'setCollateralizationOracle(address)',
+      arguments: (addresses) => [addresses.collateralizationOracle],
+      description: 'Set CR Oracle in PCV Equity Minter (buybacks) to the uncached CR Oracle'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.volt, addresses.voltOracle],
+      description: 'Replace VOLT oracle in CR oracle'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatio(address,address,uint256)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.ethPSM, '10000'],
+      description: 'Withdraw remaining ETH/FEI liquidity from Uniswap v2'
+    },
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.uniswapPCVDeposit, addresses.voltOracle],
```

Comment:
First argument here should be `addresses.volt` instead of `addresses.uniswapPCVDeposit`. But it's also a duplicate of an earlier action, so can be removed